source/includes/documentation/_vue_js_elements.md

```diff
@@ -732,45 +732,56 @@ For examples of usage see: [JDI Vuetify Sliders tests](https://github.com/jdi-te
-**getInputColor()** | Get color in RGBA format | String
-**getLabelText()** | Get label text  | String
-**getLabelHTML()** | Get label HTML element | boolean
-**isChecked()/isNotChecked()** | Shows that required element is selected/not selected | boolean
-**isDisabled()/isEnabled()** | Shows that required element is disabled/enabled | boolean
-**hasLabel()** | Shows that required element has label | boolean
-**hasInputProgressBar()** | Shows that required element has Progress bar | boolean
-
-For examples of usage see: [JDI Vuetify Lists tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SwitchTests.java).
+__Switches__ - The `v-switch` component provides users the ability to choose between two distinct values.
+
+![Switches example](../../images/vuetify/switches.png)
+
+|Method | Description                                     | Return Type
+--- |-------------------------------------------------| ---
+**check()/uncheck()** | Switch element between two states               | void
+**value()** | Get switch value                                | String
+**lable()** | Get switch label                                | Lable
+**color()** | Get switch color in RGBA format                 | String
+**backgroundColor()** | Get switch background color in RGBA format      | String
+**slotsBackgroundColor()** | Get switch slot background color in RGBA format | String
+**getMessages()** | Get switch messages                             | List<String>
+**getErrorMessages()** | Get switch error messages                       | List<String>
+**getNumberErrorMessages()** | Get the number of switch error messages         | Integer
+**getSuccessMessages()** | Get switch success messages                     | List<String>
+**isChecked()/isNotChecked()** | Check if switch is selected/not selected        | boolean
+**isEnabled()** | Check if switch is enabled                      | boolean
+**isFlat()** | Checks that switch is flat                      | boolean
+**isInset()** | Checks that switch is inset                     | boolean
+**isReadonly()** | Checks that switch is readonly                  | boolean
+**hasMessages()** | Check that switch has messages                  | boolean
+**hasErrorMessage()** | Check that switch has error messages            | boolean
+**hasSuccessMessage()** | Check that switch has success messages          | boolean
+**hasIconAppend()** | Check that switch has icon-append               | boolean
+**hasIconPrepend()** | Check that switch has icon-prepend              | boolean
+**hasRipple()** | Check that switch has ripple                    | boolean
+**is()** | Switch assert                                   | SwitchAssert
+
+For examples of usage see: [JDI Vuetify Switch tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/SwitchTests.java).
 
```

Comment:
Hi :) failed to find this point "7.	Element code (with reference to framework version - IMPORTANT!)" please, clarify